config/vendor_api/v1.4.yml

```diff
@@ -0,0 +1,38 @@
+---
+openapi: 3.0.0
+info:
+  version: v1.4
+  title: Apply API
+  contact:
+    name: DfE
+    email: becomingateacher@digital.education.gov.uk
+  description: |
+    API for DfE’s Apply for teacher training service.
+    Endpoints with the `/applications` prefix are considered stable.
+    Experimental endpoints prefixed with `/test-data` may change or be removed.
+servers:
+  - description: Sandbox (test environment for vendors)
+    url: https://sandbox.apply-for-teacher-training.service.gov.uk/api/v1.4
+  - description: Production
+    url: https://www.apply-for-teacher-training.service.gov.uk/api/v1.4
+components:
+  schemas:
+    Qualification:
+      type: object
+      additionalProperties: false
+      properties:
+        currently_completing_qualification:
+          type: boolean
+          description: The response to the “Are you currently studying to retake your GCSE?” question.
+          example: true
+          nullable: true
```

Comment:
These aren't rendering for me on the docs – unless i'm being completely blind?

Reply:
@JR-G should work fine now.

Reply:
yes – thanks!